dune-project

```diff
@@ -0,0 +1,26 @@
+(lang dune 2.0)
+
+(name gpx)
+(version 1.1.1)
+
+(generate_opam_files true)
```

Comment:
Also it seems like you missed adding the generated `gpx.opam` file to the PR. Without it `opam pin` would not work properly.

As a side note as well, i personally don’t think generating the opam file from dune should be done. It make it harder to maintain and is riddled with bugs.

Reply:
Oh, I should have tested it... thanks again!